.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "nuget"
+    directory: "/ParquetSharp.DataFrame"
```

Comment:
I don't think it makes sense to enable dependabot for ParquetSharp.DataFrame. Because this is a library, we want to maintain compatibility with older versions of packages so that end-user applications can choose the version of dependencies they want without being forced to always be on the most recent version.

Can we remove the configuration for ParquetSharp.DataFrame and only enable dependabot for the tests and GitHub actions?